### PR TITLE
Task/prompt and send invitation

### LINKED ISSUE #703: Prompt and send invite when reviewer email is not found
As part of #253

### PR DESCRIPTION
**Pull request by jseraf**
Part of #253 
Closes #703

### CODE REVIEW

**Review by mattbaumann1 — APPROVED** [review]

File: app/controllers/grant_reviewers/invitations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  class InvitationsController < ApplicationController
+    before_action :set_grant
+
+    def index
+      authorize @grant, :edit?
+      @invitations = GrantReviewer::Invitation.by_grant(@grant)
+      skip_after_action :verify_policy_scope
+    end
+
+    def update;end
```
Comment (fanglinnw) [inline app/controllers/grant_reviewers/invitations_controller.rb:11]: is this a stub to be implemented later?

Comment (jseraf) [reply app/controllers/grant_reviewers/invitations_controller.rb:11]: yep

File: app/controllers/grant_reviewers/invitations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+module GrantReviewers
+  class InvitationsController < ApplicationController
+    before_action :set_grant
+
+    def index
+      authorize @grant, :edit?
+      @invitations = GrantReviewer::Invitation.by_grant(@grant)
+      skip_after_action :verify_policy_scope
```
Comment (fanglinnw) [inline app/controllers/grant_reviewers/invitations_controller.rb:8]: should be verify_policy_scope**d**?

File: app/controllers/grant_reviewers/invitations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    end
+
+    def update;end
+
+    def create
+      authorize @grant, :edit?
+      invitation = GrantReviewer::Invitation.create(grant: @grant, inviter: current_user, email: params[:email])
+      if invitation.errors.none?
```
Comment (fanglinnw) [inline app/controllers/grant_reviewers/invitations_controller.rb:16]: there can be other type of failures such as database connection error, in addition to what .errors() returns which is specfic to activerecord validation errors.
The standard way is to do a .new() first and check if .save() returns true

**Review by fanglinnw — APPROVED** [review]

**Comment by jseraf** [comment]
wrong base branch.

**Pull request closed without merging**